api/src/controllers/resumeReview/getAllResumeReviews.ts

```diff
@@ -37,14 +44,18 @@ type ResBody = {
  * Get all resume reviews.
  * @param req HTTP request.
  * @param res HTTP response.
- * @returns HTTP response.
+ * @returns All resume reviews.
  */
-const getResumeReviews = controller(async (req: Request<unknown, ResBody, unknown, ReqQuery>, res: Response<ResBody>): Promise<void> => {
+const getAllResumeReviews = controller(async (req: Request<unknown, ResBody, unknown, ReqQuery>, res: Response<ResBody>): Promise<void> => {
     await new ReqQueryValidator().validateAndThrow(req.query);
 
-    const allResumeReviews = await resumeReviewRepository.get(req.query.reviewee, req.query.reviewer, req.query.state);
+    const id = decodeQueryToUser(req.query.id);
```

Comment:
Nit: I think we can safely `decodeQueryToUser` because
1. Id is a uuid which is URL safe (I think, [source](https://stackoverflow.com/questions/23146193/is-a-uuid-url-safe))
2. `decodeQueryToUser` implies we want a user ID out of the query field but `id` isn't really a user ID

Reply:
Yep this is a big mistake on my part. I need to change id back to being a uuid. I got mixed up when I was making all of these changes.